skimage/draw/draw.py

```diff
@@ -693,3 +693,77 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(start, end = None, extent = None, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle. Either
+    end or extent are required but not both: (extent is None) ^ (end is None)
+
+    Parameters
+    ----------
+    start : tuple
+        origin point of the rectangle, ([plane], row, column[, ...])
+    end : tuple
+        end point of the rectangle ([num_planes], num_rows, num_cols[, ...]).
+    extent : tuple
+        extent vector ([num_planes], num_rows, num_cols[, ...]) of the
+        rectangle. Will be added to start to find the end point.
```

Comment:
Sorry, I must be in some strange time warp - I just saw your comment that answers my question.